frontend/tsconfig.json

```diff
@@ -15,8 +15,8 @@
     "baseUrl": ".",
     "types": [
       "vuetify",
-      "webpack",
-      "webpack-env"
+      "vite/client",
+      "unplugin-vue2-script-setup/types"
```

Comment:
can we start migrating to `<script setup>` :smile:

Reply:
yes we can 😄

Reply:
I am sorry for barging in but I just had to

![yes-we-can-yaby2n](https://user-images.githubusercontent.com/1658405/170973748-b54d7efe-3157-4d13-9770-0084e7becb86.jpg)